git-find-lfs-extensions.py

```diff
@@ -111,3 +123,10 @@ def print_line(ext, share_large, count_large, count_all, size_all, min, max):
             int(result[ext]['min']),
             int(result[ext]['max'])
         )
+
+print("\nAdd to .gitattributes:\n")
+for ext in sorted(result, key=lambda x: (result[x]['type'], x)):
+    if len(ext) > 0 and result[ext]['type'] == "binary" and result[ext]['count_large'] > 0:
+        print('*.{} filter=lfs diff=lfs merge=lfs -text'.format(
+            "".join(c if ('0' <= c <= '9') else "[" + c.upper() + c.lower() + "]" for c in ext)
+        ))
```

Comment:
- We probably should make them lowercase in the table, I didn't notice that :)
- Special characters just didn't occur to me... is there a superset of valid characters (for various OSes/platforms/file systems) we could reference?

Honestly this functionality could probably be in git-lfs proper. I can't stand the built in track.

Also, wasn't it you who raised this issue with git? That they should really just have a preference for matching these case case-insensitively? That would be sweet.